### PR TITLE
WAS-422 Adding Airbrake (Errbit) exception monitoring

### PR DESCRIPTION
**Pull request by rickclare**
This PR adds Airbrake exception monitoring, sending any app exceptions to our Errbit server https://dst-errbit.herokuapp.com)  WAS-422

### CODE REVIEW

**Comment by Cruikshanks** [comment]
From my reading of errbit mongodb is a requirement. Is that correct? Assuming it is we will also have to consider what backup requirement for the errbit database, plus what is required server wise to run errbit within our environment, because running on heroku won't be an accepted production solution for this; it will need to be in our AWS area. On top of that do you see this being something that future services could hook into i.e. we don't have a separate errbit instance for each service? This again will have wider considerations, for example these changes may need to have gone into digital-services-core.

So in summary, its inclusion now is more a means to assess the benefits of errbit and automatic error logging and management, not a confirmation that this is going to be included in the final system. So as far as I can see this can be stripped out easily at this point and therefore is okay to merge (can you confirm this). And until those discussions have happened and that decision is made we will need to ensure this remains the case.

**Comment by rickclare** [comment]
@Cruikshanks Those are all good points.  Here are my thoughts:

* Errbit can be hosted anywhere (e.g. AWS) but you are right about requiring MongoDB.
* I don't see the need for a DB backup-solution - Exception messages shouldn't have a long lifespan, so there shoudn't be a need to back them up as the records in the database are fairly transient.
* The other option is to use a [Airbrake](https://airbrake.io) itself (a commercial solution), although I appreciate that might not suit the "in house" security requirements. Alternatively there is [Bugsnag](https://bugsnag.com/)
* I deliberately created a "dst-errbit.herokuapp.com" server (instead of "waste-exemptions-errbit.herokuapp.com", with the assumption that we could use this across DST apps.
* The change in this PR is a configuration change, specific to the Waste-exemptions app on Errbit.  Therefore, this shouldn't be required in Digital-services-core, as it is specific to Waste-excemptions.
* Yes, this is very easy to strip out (just remove the `airbrake` gem from the `Gemfile`)
* The other option is to use a [Airbrake](https://airbrake.io) itself (a commercial solution), although I appreciate that might not suit the "in house" security requirements. Alternatively there is [Bugsnag](https://bugsnag.com/)

Hope that all helps - this might be more easily discussed face-to-face.

**Comment by Cruikshanks** [comment]
Yep, agree this needs a face to face to discuss these things in more detail (will put something in the diary), but from a change point of view I'm happy for these to be merged in.
